src/__tests__/Navbar.test.js

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+import Navbar from '../components/Navbar';
+
+describe('Navbar component', () => {
+  test('renders title "CryptoMarket"', () => {
+    render(
+      <BrowserRouter><Navbar /></BrowserRouter>,
+    );
+
+    const title = screen.getByText('Cryptocurrency Market');
+    expect(title).toBeInTheDocument();
+  });
+
+  test('component match with the snapshot', () => {
+    // Assert
+    const tree = renderer.create(<BrowserRouter><Navbar /></BrowserRouter>).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
- [ ] Here is a failing test, let's fix it
<img width="843" alt="Screenshot 2023-03-16 at 10 40 39 PM" src="https://user-images.githubusercontent.com/40503704/225759009-b099030d-ed20-4063-af25-02e016136e36.png">